src/main/java/nl/tudelft/contextproject/gui/DirectorLiveController.java

```diff
@@ -123,15 +128,64 @@ private void initializeButtons() {
                 smallStatusLabel.setStyle("");
                 bigStatusLabel.setStyle("-fx-text-fill: red;");
             }
+            
+            switchLive();
+            
+            updateTables(live);
         });
 
         btnBack.toFront();
         btnBack.setOnAction((event) -> {
             MenuController.show();
         });
+        
+        btnNext.setOnAction((event) -> {
+            script.next();
+            updateTables(live);
+        });
     }
 
     /**
+     * Updates the table contents according to the current position in the script.
+     */
+    private void updateTables(boolean live) {
+        Shot smallShot;
+        Shot bigShot;
+        if (live) {
+            smallShot = script.getNextShot();
+            bigShot = script.getCurrentShot();
+        } else {
+            smallShot = script.getCurrentShot();
+            bigShot = script.getNextShot();
+        }
+        
+        if (smallShot != null) {
+            smallShotNumberLabel.setText(smallShot.getShotId());
+            smallCameraNumberLabel.setText(Integer.toString(smallShot.getCamera().getNumber()));
+            smallPresetLabel.setText(Integer.toString(smallShot.getPreset().getId()));
+            smallDescriptionField.setText(smallShot.getDescription());
+        }
+
+        if (script.getCurrentShot() != null) {
+            bigShotNumberLabel.setText(bigShot.getShotId());
+            bigCameraNumberLabel.setText(Integer.toString(bigShot.getCamera().getNumber()));
+            bigPresetLabel.setText(Integer.toString(bigShot.getPreset().getId()));
+            bigDescriptionField.setText(bigShot.getDescription());
+        }
+    }
+    
+    /**
+     * Sets live to false if it is true and to true if it is false.
+     */
+    private void switchLive() {
+        if (live) {
```

Comment:
This method can be shortened to one line: `live = live ? false : true;` Even shorter: `live = !live`